src/util/apiUtils.js

```diff
@@ -317,10 +320,22 @@ export function getAuthHeaders(request) {
 	};
 }
 
+export function getLanguageHeader(request) {
+	const cookieLang = getCookieLang(request);
+	const headerLang = request.headers['accept-language'];
+	const acceptLang = cookieLang ?
+		`${cookieLang},${headerLang}` :
```

Comment:
Only thing I can see happening here is dupe language entries, ie. something like `en-US,en-US,en`, but I'm guessing that won't matter